CMakeLists_vcpkg.txt

```diff
@@ -65,7 +66,7 @@ endif()
 #
 
 find_package(remill CONFIG REQUIRED)
-list(APPEND PROJECT_LIBRARIES remill)
+list(APPEND PROJECT_LIBRARIES remill_settings remill)
```

Comment:
Is this one of those things where we should be doing `remill::settings`?

Reply:
Probably, yeah. It's a little weird right now. Opened an issue here https://github.com/lifting-bits/remill/issues/479